modules/apps/commerce/commerce-product-test/src/testFunctional/macros/FrontStore.macro

```diff
@@ -472,7 +472,7 @@ definition {
 	}
 
 	@summary = "Default summary"
-	macro editMiniCartItemBundledProduct(date = null, boolean = null, multipleSelectionValuesChecked = null, cartItemOptionName = null, multipleSelectionValuesNotChecked = null, numeric = null, cartItemOptionValue = null, text = null, singleSelectionValueNotChecked = null, singleSelectionValueChecked = null) {
+	macro editMiniCartItemOptionProduct(date = null, boolean = null, multipleSelectionValuesChecked = null, cartItemOptionName = null, multipleSelectionValuesNotChecked = null, numeric = null, cartItemOptionValue = null, text = null, singleSelectionValueNotChecked = null, singleSelectionValueChecked = null) {
```

Comment:
The product option is being edited, right? We may name the macro like this:
**editMiniCartItemProductOption**